README.md

```diff
@@ -595,8 +595,8 @@ the specified threshold limit.
     To upgrade existing installations:
 
     *   Re-run the Terraform, to update the Cloud Functions and Scheduled Query
-    *   Update the SQL used in the Data Studio dashboard according to Step #7
-        of [3.10 Data Studio Dashboard setup](#310-data-studio-dashboard-setup).
+    *   Update the SQL used in the Looker Studio dashboard according to Step #7
+        of [3.10 Looker Studio Dashboard setup](#310-data-studio-dashboard-setup).
```

Comment:
`#310-data-studio-dashboard-setup` needs to be updated to `#310-looker-studio-dashboard-setup`

Reply:
Already fixed in the second commit

Reply:
The second commit changes it on line 82, but it also needs to be updated on line 599.

Reply:
Thanks Ben! Updated the link and all checks passed.